packages/libs/react-ui/src/components/Grid/Grid.css.ts

```diff
@@ -0,0 +1,39 @@
+import { vars } from '../../styles';
+
+import { style, styleVariants } from '@vanilla-extract/css';
+
+export const GridContainerClass = style([
```

Comment:
I'm noticing that you are capitalizing the classNames. I don't think this is normal convention. Is there a reason why you decided to do this?

Reply:
Back in stitches, similar objects contained the actual components and went capitalized. I see it makes sense to change to camel-case for the classNames